cmds/boot/fitboot/main.go

```diff
@@ -65,9 +67,18 @@ func main() {
 		kernelCmd = fmt.Sprintf("acpi_rsdp=%x %s", r.RSDPAddr(), kernelCmd)
 	}
 
-	f.Cmdline, f.Dryrun = kernelCmd, *dryRun
+	f.Cmdline = kernelCmd
 
 	if err := f.Load(*debug); err != nil {
 		log.Fatal(err)
 	}
+
+	if !*dryRun {
+		if err := boot.Execute(); err != nil {
+			log.Fatal(err)
+		}
+	} else {
```

Comment:
you don't need an else here. 

It either reboots or errors before this.

Reply:
The else is for `! dryRun` in which case it just prints not booting. Believe it's fine.

Reply:
Point being, you don't need the else.
you can just have the line 
v("Not trying to boot since this is a dry run")
after the if. B/c nothing after the if will run.

The more common we way do it.
if *dryRun {
v("...")
os.Exit(0)
}

Reply:
Ah, misunderstood the first comment then. Agree this looks cleaner. Squashed into last commit.